File: libraries/hdm/functions/types/request/ChangeDeploymentStateDTO.java
package hdm.functions.types.request;

import hdm.functions.types.struct.OperationStructDTO;
import java.io.Serializable;

public class ChangeDeploymentStateDTO
  implements Serializable
{
  private OperationStructDTO[] operations;

  public OperationStructDTO[] getOperations()
  {
    return this.operations;
  }

  public void setOperations(OperationStructDTO[] operations)
  {
    this.operations = operations;
  }
}

File: libraries/hdm/functions/types/request/ChangeDeploymentStateDTO.js
/* @org.eclipse.vjet.dsf.resource.utils.CodeGen("VjoGenerator") */
vjo.ctype('hdm.functions.types.request.ChangeDeploymentStateDTO') //< public
//> needs hdm.functions.types.struct.OperationStructDTO
.satisfies('java.io.Serializable')
.protos({
    operations:null, //< private OperationStructDTO[]
    //> public OperationStructDTO[] getOperations()
    getOperations:function(){
        return this.operations;
    },
    //> public void setOperations(OperationStructDTO[] operations)
    setOperations:function(operations){
        this.operations=operations;
    }
})
.endType();